### DISCUSSION TITLE
Can someone help my vercel has blogs.html in source but when on output blogs.html cannot be seen.

### DISCUSSION THREAD

**Discussion by im-yonghern** [vercel/community#4522] [Help]
### Summary

website: https://yonghern.dev/blogs.html

Can someone help my vercel has blogs.html in source but when on output blogs.html cannot be seen.

I've tried to change many things but all leads to the same error **404: NOT_FOUND
Code: NOT_FOUND**

### Example

https://yonghern.dev/blogs.html

### Steps to Reproduce

https://yonghern.dev/blogs.html

**Comment by amyegan** [comment]
Do you have a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) that we can use to help you with debugging?

**Reply by im-yonghern** [reply]
Not really but it worked before but it shows that error after i added supabase backend to my website

**Comment by im-yonghern** [comment]
OH Never mind I found the bug, I just had to make a vite.config.js then add the directory in there
